praw/models/reddit/emoji.py

```diff
@@ -173,12 +173,16 @@ def __iter__(self) -> List[Emoji]:
                print(emoji)
 
         """
-        response = self.subreddit._reddit.get(
+        response = self._reddit.get(
             API_PATH["emoji_list"].format(subreddit=self.subreddit)
         )
-        for emoji_name, emoji_data in response[
-            self.subreddit.fullname
-        ].items():
+        subreddit_keys = [
+            key
+            for key in response.keys()
+            if key.startswith(self._reddit.config.kinds["subreddit"])
+        ]
+        assert len(subreddit_keys) == 1
```

Comment:
This legitimately might break things for people, but I've yet to find a case where results for more than 1 subreddit are returned. I tried a `sub1+sub2` type multi, but haven't tried anything beyond that.

Reply:
What about checking for a `+`, and then splitting the subreddit into the smaller pieces, and getting their emojis, which is then joined together?

Reply:
If you find an example that actually works through this endpoint that way, then maybe, but when I tried it it didn't work.

Reply:
@bboe H4CKY54CK/praw#7

Reply:
This can be resolved @bboe